app/controllers/api/v1/users_controller.rb

```diff
@@ -13,7 +13,7 @@ def update
       private
 
       def user_params
-        params.require(:user).permit(:email, :name, :gender, :avatar)
+        params.require(:user).permit(:email, :name, :gender, :avatar, :push_token)
```

Comment:
The user could have more than one push_token, right? Because maybe he/she has more than one device